README.md

```diff
@@ -0,0 +1,130 @@
+# Parking meter REST API [![CircleCI](https://circleci.com/gh/WojciechWeg/parking_meter_rest_api.svg?style=svg)](https://circleci.com/gh/WojciechWeg/parking_meter_rest_api)
+
+Aim of this project is to build a service for managing the city parking spaces.
+
+
+### Technologies
+
+* JDK 8
+* Embeded H2 databse
+* Mapstruct 1.2.0.CR2
+* Spring boot starter data jpa
+* Spring boot starter web
+* Spring boot devtools
+* Spring boot starter test
+
+### Setup
+
+```
+chmod +x ./start.sh
+./start.sh
+```
+
+### Action table
+
+| Method     | URI                               | Action                                                							    |  Input format | Return format | 
+|------------|-----------------------------------|-------------------------------------------------------------------------|------------------|-------------------|
+| `GET` | `/start/{ticket_type}/{nr_plate}`  |  Starts parking meter with given ticket_type and nr_plate| ticket_type: String "regular"/"disabled", nr_plate: 5 char String	|Ticket           |
+| `GET`     | `/stop/{id}`                           |  Stops parking  meter/ stops ticket with  given id    		|	id: Long 			|String			|
+| `GET`      | `/check_charge/{id}`              |  Checks charge for exact ticket with given id       			|	id: Long				|String			|
+| `GET` 	|  ` /sum`								   |  Returns sum of all tickets charges									|	- 						|SumJSON		|
+| `GET`     | `/hasStarted/{nr_plate}` 		   |  Checks if car with given nr_plate started parking meter 	|	nr_plate: 5 char String |HasStartedJSON |
+
+
+## Return format
+### Start ticket
+#### Ticket
+
+Values here are:
+
+* ticketType can be REGULAR or DISABLED only
+* charge is calculated according to charge table
+* stampStart is start time
+* id  - duh
+
+```
+{
+    "ticketType": "REGULAR",
+    "charge": 0,
+    "stampStart": "2019-01-20T18:02:39.568",
+    "id": 9
+}
+```
+
+
+| Case                           | Value returned                   | HTTP status |
+|----------------------------- |----------------------------------|-----------------|
+|Fine, new ticket				| Ticket								 | 200			  |
+|Invalid nr_plate				| Null ticket						 | 400			  |
```

Comment:
co to znaczy `Null ticket`?

Reply:
Object Ticket który ma same wartości null. Nie miałem innego pomysłu jak to zrobić.